bundles/org.openhab.binding.lgthinq/README.md

```diff
@@ -0,0 +1,195 @@
+
+#### More Information
+
+This Channel Group is reports useful information data for the device:
+
+| channel          | type      | description                                                |
+|------------------|-----------|------------------------------------------------------------|
+| Fresh Air Filter | String    | Shows the Fresh Air filter status (OFF/AUTO/POWER/REPLACE) |
+| Water Filter     | String    | Shows the filter's used months                             |
+
+OBS: some versions of this device can not support all the channels, depending on the model's capabilities.
+
+**Important:** this binding will always interact with the LG API server to get information about the device. This is the Smart ThinQ way to work, there is no other way (like direct access) to the devices. Hence, some side effects will happen in the following situations:
+1. **Internet Link** - if you OpenHab server doesn't have a good internet connection this binding will not work properly! In the same way, if the internet link goes down, your Things and Bridge going to be Offline as well, and you won't be able to control the devices though OpenHab until the link comes back.
+2. **LG ThinQ App** - if you've already used the LG ThinQ App to control your devices and hold it constantly activated in your mobile phone, you may experience some instability because the App (and Binding) will try to lock the device in LG ThinQ API Server to get it's current state. In the app, you may see some information in the device informing that "The device is being used by other" (something like this) and in the OpenHab, the thing can go Offline for a while.
+3. **Pooling time** - both Bridge and Thing use pooling strategy to get the current state information about the registered devices. Note that the Thing pooling time is internal and can't be changed (please, don't change in the source code) and the Bridge can be changed for something greater than 300 seconds, and it's recommended long pooling periods for the Bridge because the discovery process fetch a lot of information from the LG API Server, depending on the number of devices you have registered in your account. 
+About this last point, it's important to know that LG API is not Open & Public, i.e, only LG Official Partners with some agreement have access to their support and documentations. This binding was a hard (very hard actually) work to dig and reverse engineering in the LG's ThinQ API protocol. Because this, you must respect the hardcoded pool period to do not put your account in LG Blacklist.
+
+## Be nice!
+If you like the binding, why don't you support me by buying me a coffee?
+It would certainly motivate me to further improve this work or to create new others cool bindings for OpenHab !
+
+[![Buy me a coffee!](https://www.buymeacoffee.com/assets/img/custom_images/black_img.png)](https://www.buymeacoffee.com/nemerdaud)
```

Comment:
I won't be mad about having to remove this. In fact, I don't need my coffees to be paid for :-D and the ones I received I converted to charity. But I believe this is a good practice so that opensource supporters are somehow more motivated and can grow.